cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -0,0 +1,135 @@
+  zipkin:
+    endpoint: 0.0.0.0:9411
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+    listenAddress: 0.0.0.0:9080
+  smartagent/ecs-metadata:
+    type: ecs-metadata
+    metadataEndpoint: "${ECS_TASK_METADATA_ENDPOINT}"
+    statsEndpoint: "${ECS_TASK_STATS_ENDPOINT}"
+    excludedImages: ${env:ECS_METADATA_EXCLUDED_IMAGES}
+
+processors:
+  batch:
+  # Enabling the memory_limiter is strongly recommended for every pipeline.
+  # Configuration is based on the amount of memory allocated to the collector.
+  # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+  # should be 90% of the collector's memory. The simplest way to specify the
+  # ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable. Alternatively, the
+  # --mem-ballast-size-mib command line flag can be passed and take priority.
+  # For more information about memory limiter, see
+  # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiter/README.md
+  memory_limiter:
+    check_interval: 2s
+    limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
+  # detect if the collector is running on a cloud system
+  # important for creating unique cloud provider dimensions
+  resourcedetection:
+    detectors: [ecs]
+    override: false
+  # Enables the filter processor with example settings
+  # Full configuration here: https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/filterprocessor
+  # NOTE: These settings need to be change when using this processor
+  filter/1:
+    metrics:
+      exclude:
+        match_type: regexp
+        metric_names: ${env:METRICS_TO_EXCLUDE}
+        # Optional: The following processor can be used to add a default "deployment.environment" attribute to the logs and
+        # traces when it's not populated by instrumentation libraries.
+        # If enabled, make sure to enable this processor in the pipeline below.
+        #resource/add_environment:
+        #attributes:
+        #- action: insert
+        #value: staging/production/...
+        #key: deployment.environment
+
+exporters:
+  # Traces
+  sapm:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
+  # Metrics + Events
+  signalfx:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    realm: "${SPLUNK_REALM}"
+    correlation:
+#  # Logs
+#  splunk_hec:
+#    token: "${SPLUNK_HEC_TOKEN}"
+#    endpoint: "${SPLUNK_HEC_URL}"
+#    source: "otel"
+#    sourcetype: "otel"
+
+service:
+  extensions: [health_check, http_forwarder, zpages]
+  pipelines:
+    traces:
+      receivers: [jaeger, otlp, zipkin, smartagent/signalfx-forwarder]
+      processors:
+        - memory_limiter
+        - batch
+        - resourcedetection
+      #        - resource/add_environment
+      exporters: [sapm, signalfx]
+    metrics:
+      receivers: [signalfx, smartagent/signalfx-forwarder, smartagent/ecs-metadata, prometheus/internal]
```

Comment:
I believe internal metrics need to be processed in a different pipeline with different configuration -- at least this is the case for helm

Reply:
Ok, I moved internal metrics to a separate pipeline.